reports/TC39/2017-01.md

```diff
@@ -0,0 +1,124 @@
+a student from the queer community who works with the Node.js team. This is the first meeting that had any women in it,
+so it was great that there could in fact be three of us.
+
+## On the Subject of Diversity
+
+The adoption of the new Code of Conduct for TC39 proceeds forward. This code of conduct is based off of the JS Foundation Code of Conduct
+and has been proven to work before. It is being put in place in hopes of making TC39 a more welcoming community for all.
+Of major concern was how code of conduct violations would be enforced. There was general consensus that any body investigating a code of conduct violation
+would need to be made up of a fairly diverse set of persons, some who were present when the violation occurred and some who were not. It was also decided that
+given the rarity of code of conduct violations, investigation bodies could be formed at the time of complaint, and there would not have to be one on call at all times.
+
+TC39 members are positive about moving forward with the code of conduct, and are encouraged to review and make comments before the March meeting.
+[Community feedback on the new code of conduct is welcome](https://github.com/tc39/code-of-conduct-proposal).
+
+## Getting Back to The Code
+
+[You can find a full agenda of what was discussed at this meeting here.](https://github.com/tc39/agendas/blob/master/2017/01.md) I won't discuss every single line item
+but I want to highlight a few proposals that are potentially of great benefit. In particular, I want to touch upon:
++ Null Propagation Operator
++ RegExp Lookbehind
++ 64 Bit Integers
++ IEEE 754 Sign Bit
+
+
+### Null Propagation Operator
+
+For the average developer, the `?.` operator - called here null propagation, but in other languages called null conditional, safe navigation, or elvis,
+is probably the most exciting proposal that moved forward at this meeting.
+It's use can be expressed easily by this code from [the proposal](https://docs.google.com/presentation/d/11O_wIBBbZgE1bMVRJI8kGnmC6dWCBOwutbN9SWOK0fU/view#slide=id.g1c161255c9_0_55);
+
+Without the null propagation operator:
+
+```js
+const firstName = (message
+  && message.body
+  && message.body.user
+  && message.body.user.firstName) || 'default';
+```
+
+With the null propagation operator:
+```js
+const firstName = message.body?.user?.firstName || 'default';
+```
+
+As you can see, the null propagation operator avoids constant checks for falsiness before navigating down an object graph, saving the developer substantial effort.
+As of right now, expect the null propagation operator to return `null` if it reaches a `null` property, and `undefined` if it reaches an `undefined` property.
+
+This proposal advanced to stage 1 with overwhelming support from the committee given it's usefulness. That said, there is substantial concern about the behavior of this
+operator in several edge cases. This is a common complexity when adding syntax to a language.
+
+### RegExp Lookbehind
```

Comment:
Lookbehind got stage 2 in [December](https://github.com/tc39/proposal-regexp-lookbehind/commit/b0fc1b2726abb43c99366c4a0ca3a0be9246cf87) - Named Capture Groups is the one that [hit stage 2](https://github.com/tc39/proposals/commit/8bca71cbe127fcba6b40c2d4e97d2c75f2ebf3ed) this meeting.